addons/website_sale/tests/__init__.py

```diff
@@ -12,3 +12,4 @@
 from . import test_website_sale_image
 from . import test_website_sequence
 from . import test_website_sale_visitor
+from . import test_website_sale_product
```

Comment:
ABC, if I may :) 
Don't r- but maybe for next time this file is edited :)

Reply:
Indeed :)

As it was already not in alphanumeric order at all, I left it as is in stable.